### PR TITLE
Add  StackStorm in services 

### PR DESCRIPTION
**Pull request by thecoder8890**
Built by CA Technologies, Jarvis is another open source alternative to Hubot, specifically designed for working with build messages to make the most of CA’s Flowdock, a chat and collaboration platform. Jarvis works on the JVM, so plugins can be created in Groovy, Java, and Clojure, with support for other JVM languages possible as well such as Kotlin. The downside is extensibility — Jarvis only behaves with Flowdock.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -189,6 +191,10 @@ These open source projects will help you to build a bot to automate your company
 
 * [Hubot](https://hubot.github.com/) + [adapter](https://github.com/smoak/hubot-jabbr) (CoffeeScript, Node.js)
 
+### Jarvis
+
+* [A Flowdock robot similar to Hubot](https://github.com/RallySoftware/jarvis)
```
Comment (exAspArk) [inline README.md]: Hey, thank you for opening the PR!

It seems that Jarvis is not actively maintained right now. The last commit is from 2014

File: README.md [hunk 2]
```diff
@@ -249,6 +255,10 @@ These open source projects will help you to build a bot to automate your company
 * [Slacker](https://github.com/koudelka/slacker) (Elixir)
 * [Yetibot](https://github.com/yetibot/yetibot) (Clojure)
 
+### StackStorm
+* [StackStorm](https://github.com/StackStorm/)(node + hubot)
```
Comment (exAspArk) [inline README.md]: Does it have a Hubot adapter? If so, would you be able to specify a link:

```
[Hubot](https://hubot.github.com/) + [adapter](LINK_TO_STACK_STORM_ADAPTER) (CoffeeScript, Node.js)
```

If it is a ChatOps service (not a framework or a library), you can add it to the `Services` section

Comment (thecoder8890) [reply README.md]: Sure will do

Comment (thecoder8890) [reply README.md]: Resolved as per comment  check this commit. 
29139cc059f74168c2b712a893887bed7176ffb6

**Comment by thecoder8890** [comment]
Any Update on this pull request

File: README.md [hunk 3]
```diff
@@ -62,6 +62,7 @@ If you want to contribute, please read [CONTRIBUTING.md](./CONTRIBUTING.md).
 ## Services
 
 * [WorkflowCI](https://www.workflowci.com) – IFTTT for developers (freemium). Integrates with Slack, GitHub, CircleCI, Google Cloud Build.
+* [StackStorm](https://github.com/StackStorm/) - StackStorm Chatops in a form of packages and docker image. The package includes hubot, hubot-stackstorm and pre-installed adapters for many Chat services.
```
Comment (exAspArk) [inline README.md:65]: could we specify that it is a self-hosted platform + which chat providers it supports? e.g.

> self-hosted platform for integration and automation across services and tools. Integrates through Hubot with Slack, Microsoft Teams, Mattermost, Rocket.Chat, Cisco Spark.

P.S. there is a conflict against the latest master

**Pull request closed without merging**